hash/Cargo.toml

```diff
@@ -17,8 +17,9 @@ rustdoc-args = ["--cfg=docsrs"]
 
 [features]
 borsh = ["dep:borsh", "std"]
-bytemuck = ["dep:bytemuck", "dep:bytemuck_derive"]
-default = ["std"]
+bytemuck = ["copy", "dep:bytemuck", "dep:bytemuck_derive"]
+copy = []
+default = ["copy", "std"]
```

Comment:
Since this is a breaking change, could we remove the default features and both `"std"` and `"copy"` have to be explicitly enabled? I am thinking that people might want to have a `Hash` field on an account, so it could be useful to have the crate "lighter" by default.